tests/cross/src/test/scala/tests/pc/CompletionScalaCliSuite.scala

```diff
@@ -40,10 +40,10 @@ class CompletionScalaCliSuite extends BaseCompletionSuite {
 
   check(
     "version",
-    """|//> using lib "io.circe::circe-core_native0.4:@@"
+    """|//> using lib "io.circe::circe-core_sjs1:0.14.1@@"
```

Comment:
There is no js version for 0.13 so instead I put in a full version. Next should be 0.15 so we should safe here.